lib/flat_search/flats/flat.ex

```diff
@@ -0,0 +1,54 @@
+defmodule FlatSearch.Flats.Flat do
+  @moduledoc """
+  Flat module. Schema and changesets
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "flats" do
+    field :unique_id, :string
+    field :title, :string
+    field :link, :string
+    field :price, :integer
+    field :additional_price, :integer
+    field :negotiation, :boolean, default: false
+    field :surface, :integer
+    # field :description, {:array, :string}
```

Comment:
commented out code

Reply:
Yeah, this one is on purpose - I have to come up with a solution to this one (description comes with a weir type and is not always allowed by the changeset) but it's not important for application functionality a this point.